src/components/PivotTable/PivotTableBoard/utils.test.tsx

```diff
@@ -0,0 +1,145 @@
+import { BoardField, FieldFiltersByKey, FieldValuesByKey } from './model'
+import { getInitialKeys, handleTagFilterRemove, initializeActiveFilters } from './utils'
+
+type Test = {
+  name: string
+  size?: string
+}
+
+describe('getInitialKeys', () => {
+  it('should return initial keys based on initialFields', () => {
```

Comment:
I think is good to create a test case with availableKeys filled too when initialFields are provided